CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Breaking changes
 
 - messages produced by `cmd/bnscli` have a new binary format incompatible with
   the previous version.
+- cash.UpdateConfigurationMsg requires Metadata.Schema
```

Comment:
You can use ` (backtick) to apply code formatting and make both first and last word look nicer